source/features/github-actions-indicators.tsx

```diff
@@ -43,18 +68,39 @@ const getWorkflowsDetails = cache.function('workflows', async (): Promise<Record
 		}
 	`);
 
-	const workflows = workflowFiles?.entries ?? [];
-	if (workflows.length === 0) {
+	const workflows: [any] = workflowFiles?.entries ?? [];
```

Comment:
```suggestion
	const workflows: any[] = workflowFiles?.entries ?? [];
```

`[any]` means "array with one element of type any"

Reply:
Sorry for my bad TS skill 🥲